internal/jimm/juju/jimm.go

```diff
@@ -0,0 +1,269 @@
+// Copyright 2025 Canonical.
+
+package juju
+
+import (
```

Comment:
could we preserve the history of this file? e.g. https://stackoverflow.com/questions/16937359/git-copy-file-preserving-history

Reply:
Should be done now, curious if it worked as expected....

Reply:
On Github the file still looks "all new".